addons/upgrade/to-10.0-packetfence-pki-migrate.pl

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/perl
+
+use DBI;
+use strict;
+
+BEGIN {
+    use constant INSTALL_DIR => '/usr/local/pf';
+    use lib INSTALL_DIR . "/lib";
+}
+
+use pf::db;
+use WWW::Curl::Easy;
+
+my $driver   = "SQLite";
+my $database = "db.sqlite3";
```

Comment:
Are you sure this is the right path for the DB ?

To me it sounds like it would take db.sqlite3 in my cwd which may not be where the PKI DB is